src/core/opamJson.ml

```diff
@@ -27,7 +27,8 @@ let adds_esc b s =
       match String.get s i with
       | '"' -> flush b start i; adds b "\\\""; loop next next
       | '\\' -> flush b start i; adds b "\\\\"; loop next next
-      | '\x00' .. '\x1F' | '\x7F' (* US-ASCII control chars *) as c ->
+      | '\x00' .. '\x1F' | '\x7F' (* US-ASCII control chars *)
+      | '\x80' .. '\xFF' (* Extended ASCII chars *) as c ->
```

Comment:
You can't do this here, this will break UTF-8 output. The module assumes strings are UTF-8 encoded. 

Btw. could you please make one PR per issue you fix.